docs/integration/alerts-twilio-integration.md

```diff
@@ -0,0 +1,44 @@
+title: Twilio Alerts Integration
+description: Sematext threshold, anomaly and / or heartbeat Alerts integration with Twilio. Use them together to get the information about each notification quickly and on any device.
+
+## In Telegram
+
+**1.** Create a Twilio account at [twilio.com](https://twilio.com).
+
+**2.** Go to your Twilio project that you want to send notifications to and setup a new Twilio **Phone number**. This number will be used as the sender or the SMS messages. If you are testing Twilio you can click the **Get a Trial Number** button to generate a temporary phone number for tests. In addition to the phone number you will also need the **Account SID** and the **Auth Token** to be able to setup the notification in Sematext. Note those down:
+
+<img class="content-modal-image" alt="Create Twilio Integration - Credentials" src="../../images/integrations/create-twilio-integration-create-phone-number.png" title="Create Twilio Integration - Credentials">
+
+If you wish to test Sematext notifications in Twilio test environment head to **Settings** and its **General** subsection to get the test **Account SID** and the **Auth Token**.
+
+<img class="content-modal-image" alt="Create Twilio Integration - Test Credentials" src="../../images/integrations/create-twilio-integration-settings.png" title="Create Twilio Integration - Test Credentials">
+
+**3.** If you didn't have a Twilio phone number earlier clicking the **Get a Trial Number** button will result in displaying your phone number. Click the **Choose this Number** to accept it:
```

Comment:
will result in displaying ==> will display

Reply:
Done.